<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>C_frm_robot_calibration</class>
 <widget class="QMainWindow" name="C_frm_robot_calibration">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1782</width>
    <height>857</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTextEdit" name="txb_zaehler">
    <property name="geometry">
     <rect>
      <x>1620</x>
      <y>690</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_exit">
    <property name="geometry">
     <rect>
      <x>1620</x>
      <y>730</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_save_pose">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>730</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Calibration</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_camera_img">
    <property name="geometry">
     <rect>
      <x>1180</x>
      <y>10</y>
      <width>561</width>
      <height>581</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QSpinBox" name="num_camID">
    <property name="geometry">
     <rect>
      <x>93</x>
      <y>690</y>
      <width>51</width>
      <height>26</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_camID">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>700</y>
      <width>71</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Camera ID</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="grpb_pose_cam">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>10</y>
      <width>571</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>Pose Kamera zu Board</string>
    </property>
    <widget class="QTextEdit" name="txb_px_cam">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>40</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="txb_nx_cam">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="txb_ox_cam">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>40</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="txb_ax_cam">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>40</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="txb_ny_cam">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="txb_oy_cam">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>80</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="txb_ay_cam">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>80</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="txb_py_cam">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>80</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="txb_nz_cam">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="txb_oz_cam">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>120</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="txb_az_cam">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>120</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="txb_pz_cam">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>120</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grpb_pose_robot">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>210</y>
      <width>571</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>Pose Robot TCP zu Base</string>
    </property>
    <widget class="QTextEdit" name="txb_px_robot">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>40</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="txb_nx_robot">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="txb_ox_robot">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>40</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="txb_ax_robot">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>40</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="txb_ny_robot">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="txb_oy_robot">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>80</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="txb_ay_robot">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>80</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="txb_py_robot">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>80</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="txb_nz_robot">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="txb_oz_robot">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>120</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="txb_az_robot">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>120</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="txb_pz_robot">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>120</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="grpb_pose_robot_to_world">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>410</y>
      <width>571</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>Pose Robot Base zu Welt</string>
    </property>
    <widget class="QTextEdit" name="txb_px_robot_world">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>40</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="txb_nx_robot_world">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="txb_ox_robot_world">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>40</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="txb_ax_robot_world">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>40</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="txb_ny_robot_world">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="txb_oy_robot_world">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>80</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="txb_ay_robot_world">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>80</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="txb_py_robot_world">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>80</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="txb_nz_robot_world">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="txb_oz_robot_world">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>120</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="txb_az_robot_world">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>120</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="txb_pz_robot_world">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>120</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QPushButton" name="bt_set_home">
    <property name="geometry">
     <rect>
      <x>180</x>
      <y>730</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Set home</string>
    </property>
   </widget>
   <widget class="QPushButton" name="bt_move_home">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>730</y>
      <width>141</width>
      <height>71</height>
     </rect>
    </property>
    <property name="text">
     <string>Move to home</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="grpb_home_pose">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>571</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>Pose Home</string>
    </property>
    <widget class="QTextEdit" name="txb_px_home">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>40</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="txb_nx_home">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>40</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="txb_ox_home">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>40</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="txb_ax_home">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>40</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="txb_ny_home">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="txb_oy_home">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>80</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="txb_ay_home">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>80</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="txb_py_home">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>80</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="txb_nz_home">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>120</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="txb_oz_home">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>120</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="txb_az_home">
     <property name="geometry">
      <rect>
       <x>290</x>
       <y>120</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QTextEdit" name="txb_pz_cam_2">
     <property name="geometry">
      <rect>
       <x>430</x>
       <y>120</y>
       <width>121</width>
       <height>31</height>
      </rect>
     </property>
     <property name="readOnly">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QDoubleSpinBox" name="dspb_Panda_Vel_max">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>220</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="maximum">
     <double>1.700000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>0.200000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="dspb_Panda_Omega_max">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>320</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="maximum">
     <double>2.500000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>0.200000000000000</double>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="dspb_Panda_Acc_max">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>270</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="maximum">
     <double>13.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Panda_Alpha_max">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>366</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Panda α_max [rad/s²]</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="dspb_Panda_Alpha_max">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>370</y>
      <width>91</width>
      <height>26</height>
     </rect>
    </property>
    <property name="decimals">
     <number>1</number>
    </property>
    <property name="maximum">
     <double>25.000000000000000</double>
    </property>
    <property name="singleStep">
     <double>0.100000000000000</double>
    </property>
    <property name="value">
     <double>1.000000000000000</double>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Panda_Vel_max">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>216</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Panda v_max [m/s]</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Panda_Acc_max">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>266</y>
      <width>191</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Panda a_max [m/s²]</string>
    </property>
   </widget>
   <widget class="QLabel" name="lbl_Panda_Omega_max">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>316</y>
      <width>181</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>Panda ω_max [rad/s]</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1782</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
